modules/localization/msf/local_tool/local_visualization/online_visual/online_visualizer_component.h

```diff
@@ -91,6 +91,10 @@ class OnlineVisualizerComponent final
   std::shared_ptr<cyber::Reader<LocalizationEstimate>>
       fusion_local_listener_ = nullptr;
   std::string fusion_local_topic_ = "";
+
+  std::shared_ptr<cyber::Reader<LocalizationEstimate>>
+      ndt_local_listener_ = nullptr;
+  std::string ndt_local_topic_ = "";
```

Comment:
The default constructor of string forms an empty one.

Reply:
It will be modified in other pull request.